PVB Linx Groep 6/Assets/Code/Scripts/Enemy/Enemy.cs

```diff
@@ -26,7 +26,7 @@ public class Enemy : MonoBehaviour
         [SerializeField] private int delay;
         [SerializeField] private int damage;
         
-        private Transform _currentTarget;
+        [SerializeField] private Transform _currentTarget;
```

Comment:
Why does this need to be Serialized?

Reply:
oh, dat was eingenlijk om het te testen, was het vergeten om dat te verwijderen.